### DISCUSSION TITLE
How to calculate angle in degrees between two points ?

### DISCUSSION THREAD

**Discussion by yanc0** [duckdb/duckdb-spatial#418] [Q&A]
Hi folks !

I was wondering if anyone has calculated headings in degrees between two points on a line ?

```json
"geometry": {
  "type": "LineString",
  "coordinates": [
    [-0.2354821, 49.2847621],
    [-0.2354258, 49.2848422]
  ]
}
```

It seems that [ST_Azimuth](https://postgis.net/docs/ST_Azimuth.html) should do the trick but it's not available in spatial extension (yet ?).

**Comment by bradh** [comment]
Looks like it was added in https://github.com/duckdb/duckdb-spatial/pull/596
